bin/i18n/resources/pegasus/emails/sync_in.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/pegasus_markdown'
+require_relative '../emails'
+
+module I18n
+  module Resources
+    module Pegasus
+      module Emails
+        class SyncIn < I18n::Utils::SyncInBase
+          LOCALIZABLE_FILE_SUBPATHS = %w[
+            hoc_signup_2023_receipt_en.md
+          ].freeze
+
+          def execute
+            progress_bar.start
+
+            I18nScriptUtils.process_in_threads(LOCALIZABLE_FILE_SUBPATHS) do |file_subpath|
+              origin_file_path = File.join(ORIGIN_DIR_PATH, file_subpath)
+              next unless File.exist?(origin_file_path)
+
+              i18n_source_file_path = File.join(I18N_SOURCE_DIR_PATH, file_subpath)
+              i18n_source_file_path = i18n_source_file_path.sub('public/', '')
+
+              I18nScriptUtils.copy_file(origin_file_path, i18n_source_file_path)
+              I18n::Utils::PegasusMarkdown.sanitize_file_header(i18n_source_file_path)
+            ensure
+              mutex.synchronize {progress_bar.increment}
+            end
+
+            progress_bar.finish
+          end
+
+          def process
+            LOCALIZABLE_FILE_SUBPATHS.each do |file|
+              I18nScriptUtils.copy_file(`#{ORIGIN_DIR_PATH}/#{file}`, I18N_SOURCE_DIR_PATH)
+            end
+          end
```

Comment:
If I'm not mistaken, you only need the method process (Or at least, a public method called process).
You can add private methods to do more specific tasks and then call those methods inside `process`.
i.e.
```
def process
    prepare_content
    redact_content
    other_task
end

private

def prepare_content
    ...
end

def redact_content
    ...
end

def other_task
    ...
end
```
Sorry, it's a little bit confusing, since we optimized the template midway.